main.tex

```diff
@@ -233,9 +233,9 @@ \subsection*{\textit{Vgsc} non-synonymous nucleotide variation}
 %% L995F, L995S
 The two known resistance alleles affecting codon 995 had the highest overall allele frequencies within the Ag1000G phase 1 cohort.
 %
-The \texttt{L995F} allele was at high frequency in populations of both species from West, Central and Southern Africa.
+The \texttt{L995F} allele was at high frequency in populations of both species from West, Central and Southern Africa (Table \ref{table:variants_missense}).
```

Comment:
Rather than repeat the table citation in two sentences, suggest to move it up to the first sentence in the paragraph.

Reply:
Sure, agreed.